src/main/java/eiteam/esteemedinnovation/hammer/ContainerSteamAnvil.java

```diff
@@ -163,7 +163,7 @@ public void updateRepairOutput() {
             Enchantment enchantment;
 
             if (itemstack2 != null) {
-                flag = itemstack2.getItem() == Items.ENCHANTED_BOOK && Items.ENCHANTED_BOOK.getEnchantments(itemstack2).tagCount() > 0;
+                flag = itemstack2.getItem() == Items.ENCHANTED_BOOK && !EnchantmentHelper.getEnchantments(itemstack2).isEmpty();
```

Comment:
Hm. It might be worth the readability to add a `hasEnchantments` function to `ItemStackUtility`. Not expecting that for this PR.